Readme.md

```diff
@@ -327,6 +328,24 @@ next(3)
 completed
 ```
 
+#### apply
+
+Apply allows you to chain your own custom Observable operator-like functions
+without having to extend ObservableType. For additional rationale for this see
+[discussion on github](https://github.com/RxSwiftCommunity/RxSwiftExt/issues/73)
```

Comment:
I would perhaps be more "generic" in the description, with something like "Apply provides a unified mechanism for applying transformations on Observable sequences, without having to extend ObservableType or repeating your transformations", or something of the sorts? 

Just because "operator-like functions" perhaps seems confusing and non-descriptive necessarily, and (it could just be a closure and not necessarily a function, for example)